src/pages/in/invoice-software.js

```
@@ -900,34 +1009,38 @@ const OldInvoiceSoftware = (path) => {
+                    >
                       Run your business with automation!
                     </h2>
                     <p>
                       From setting payment reminders to sending{" "}
                       <a
-                        href='https://giddh.com/blog/benefits-of-online-invoicing-for-small-business/'
-                        target='_blank'
-                        className='text-decoration-underline'
+                        href="https://giddh.com/blog/benefits-of-online-invoicing-for-small-business/"
+                        target="_blank"
+                        className="text-decoration-underline"
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)